Bug: installation script fails on macOS (Intel)

**Describe the bug**

Unable to install Spice on macOS (intel x86_64) using quickstart instructions in repo home. I'm using GitHub Actions `macos-12` runner as test platform 

```
uname -m
x86_64
```

**To Reproduce**

1. Run `uname -m` to ensure test platform is macOS `x86_64`
2. Run `curl https://install.spiceai.org/ | /bin/bash`
4. See error **No prebuilt binary for darwin-x86_64**

```
Run curl https://install.spiceai.org/ | /bin/bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  7020  100  7020    0     0  17570      0 --:--:-- --:--:-- --:--:-- 17550
100  7020  100  7020    0     0  17563      0 --:--:-- --:--:-- --:--:-- 17550
No prebuilt binary for darwin-x86_6[4](https://github.com/sgrebnov/spiceai/actions/runs/8350214839/job/22856108684#step:4:5)
Failed to install Spice CLI
For support, see https://docs.spiceai.org/
Error: Process completed with exit code 1.
```


**Expected behavior**
The Spice is installed successfully

**Screenshots**
![image](https://github.com/spiceai/spiceai/assets/981580/e91411af-5b24-48cf-b43b-3ed40e6895a0)

**Additional notes**
Installation on macOS (Apple Silicon) works

--- a/install/install.sh
+++ b/install/install.sh
@@ -43,7 +43,7 @@ getSystemInfo() {
 }
 
 verifySupported() {
-    local supported=(linux-x86_64 darwin-aarch64)
+    local supported=(linux-x86_64 darwin-aarch64 darwin-x86_64)
     local current_osarch="${OS}-${ARCH}"
 
     for osarch in "${supported[@]}"; do

